net/libproxy/Portfile

```diff
@@ -18,94 +18,53 @@ long_description    Libproxy exists to answer the question: Given \
                     a network resource, how do I reach it? It handles all \
                     the details, enabling you to get back to programming.
 
-checksums           rmd160  18680c62196d6b1765864ac8d8cfb0da19e96fb3 \
-                    sha256  712f5965b57dc329fd5700f906d9943dbdc379e547e31dc8b11c77332a652d52 \
-                    size    98063
+checksums           rmd160  35c1118e961666ca15726ac9ffa899f30cd7355d \
+                    sha256  aed5d3ce5cfbf394595d79c717b04b61a6956540366bcbd0a73e2ae77ccfedea \
+                    size    58147
 
-patchfiles-append   patch-bindings-perl-src-CMakeLists.txt.diff \
-                    patch-libproxy-cmake.diff \
-                    patch-libproxy-test-CMakeLists.txt.diff
-
-depends_build-append \
-                    path:bin/pkg-config:pkgconfig
-
-depends_lib-append  path:bin/vala:vala \
-                    path:lib/pkgconfig/glib-2.0.pc:glib2
+depends_lib-append  \
+                    port:icu \
+                    port:gsettings-desktop-schemas \
+                    port:gobject-introspection
```

Comment:
And this as well.

Reply:
Ports that use gobject introspection typically do so by including the gobject_introspection-1.0 portgroup to handle various common problems, however I don't know if the portgroup has been updated for compatibility with meson build systems. If it hasn't, it would be a good idea to update it, and then include it in this port.

Reply:
It may be too much of a burden to put onto the PR author. This port builds fine without that PG, so probably PG can be handled separately (there are a lot of ports depending on GO without adding its PG, including meson-using ones).